mock-core-configs/mock-core-configs.spec

```diff
@@ -18,7 +18,7 @@ BuildArch:  noarch
 Provides: mock-configs
 
 # distribution-gpg-keys contains GPG keys used by mock configs
-Requires:   distribution-gpg-keys >= 1.63
+Requires:   distribution-gpg-keys >= 1.64
```

Comment:
Do we need to bump this again?

Reply:
Yes, sorry for being too fast on the gpg key side...  We want the rawhdie => F37 symlink there.